### PR TITLE
fix download hugo url err

### PR DESCRIPTION
**Pull request by JackyYangPassion**
fix download hugo url err

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -18,22 +18,22 @@ U should ensure NPM & Hugo binary [download url](https://github.com/gohugoio/hug
...
 # Linux version (0.95 extend)
-wget https://github.do/https://github.com/gohugoio/hugo/releases/download/v0.95.0/hugo_extended_0.95.0_Linux-64bit.tar.gz
+wget https://github.com/gohugoio/hugo/releases/download/v0.95.0/hugo_extended_0.95.0_Linux-64bit.tar.gz
 
 # 解压后 hugo 是单二进制文件, 可直接使用, 或推荐放 /usr/bin 及环境变量下.
 sudo install hugo /usr/bin # 如果 mac 提示没有权限, 你可以直接使用它, 也可以 mv hugo /usr/bin 代替
 
 # 1. download website's source code
-git clone https://github.com/hugegraph/hugegraph-doc.git website
+git clone https://github.com/hugegraph/hugegraph-doc.git master
```
Comment (imbajin) [inline contribution.md:30]: why use `master` as the directory name?

File: contribution.md [hunk 2]
```diff
@@ -18,22 +18,22 @@ U should ensure NPM & Hugo binary [download url](https://github.com/gohugoio/hug
 # 0. install npm & hugo if you don't have it
 
 # Mac version (0.95 extend)
-wget https://github.do/https://github.com/gohugoio/hugo/releases/download/v0.95.0/hugo_extended_0.95.0_macOS-64bit.tar.gz
```
Comment (imbajin) [inline contribution.md:21]: thanks, the CDN site seems broken now... maybe we could add a CDN URL later

File: contribution.md (hunk 1 above)
Comment (JackyYangPassion) [reply contribution.md]: Understand it as -b website branch ，Thanks for the reminder

File: contribution.md (hunk 2 above)
Comment (JackyYangPassion) [reply contribution.md:21]: ok

**Review by imbajin — APPROVED** [review]

**Pull request merged**